Cargo.toml

```diff
@@ -4,7 +4,7 @@ members = ["near-sdk", "near-sdk-macros", "near-contract-standards", "near-sys"]
 exclude = ["examples/"]
 
 [workspace.package]
-version = "5.7.1"
+version = "6.0.0"
```

Comment:
me or someone else will have to patch it down to 5.8.0 manually in `release-plz-2025-02-03T20-41-52Z` branch to hush-hush `cargo-semver-checks` a little and